README.md

```diff
@@ -114,6 +114,16 @@ and it also has some options and third-level commands.
 | `--start` | See [Common options](#common-options) | See [Common options](#common-options) |
 | `--end` | See [Common options](#common-options) | See [Common options](#common-options) |
 
+### `metrics` second-level command
+`metrics` second-level command is an entrance for all operations related to metrics,
```

Comment:
@kezhenxu94 Document change is missing, please recheck whether there are other places.